clojure-skeleton/src/ant_battles/core.clj

```diff
@@ -0,0 +1,26 @@
+(ns ant-battles.core
+  (:require [ant-battles.symbolizer :as sym]
+            [clj-http.client :as client]
+            [clj-json [core :as json]]))
+
+(defn- from-json [data]
+  (sym/symbolize (json/parse-string data)))
+
+(defn- read-json [url]
+  (from-json
+    (get (client/get url) :body)))
+
+(defn index [base]
```

Comment:
I don't see an easy way to get rid of the base variable, here's what I can think of in order of preference:

* Use [profiles](https://github.com/technomancy/leiningen/blob/master/doc/PROFILES.md) to configure the ant server endpoint.
* Hardcode it: Put `(def base "localhost:4000")` at the top of this file. In tests when you need to talk to a different server you can use `with-redefs` to override it.
* Use an atom to have some mutable global state that you can swap out in tests.